Description: Disable fft2d because not in Debian due to license issue.
 See https://salsa.debian.org/deeplearning-team/oourafft
Forwarded: not-needed
Author: Dylan Aïssi <daissi@debian.org>

--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -146,7 +146,6 @@
 find_package(absl REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(farmhash REQUIRED)
-find_package(fft2d REQUIRED)
 find_package(FlatBuffers REQUIRED)
 find_package(gemmlowp REQUIRED)
 if (NOT CMAKE_SYSTEM_PROCESSOR OR CMAKE_SYSTEM_PROCESSOR MATCHES "x86")
@@ -617,7 +616,6 @@
     absl::synchronization
     absl::variant
     farmhash
-    fft2d_fftsg2d
     flatbuffers::flatbuffers
     gemmlowp::gemmlowp
     ruy::ruy
